pallene/checker.lua

```diff
@@ -407,10 +429,13 @@ function FunChecker:expand_function_returns(lhs, rhs)
     end
 end
 
-
 function FunChecker:check_function(lambda, func_typ)
     assert(lambda._tag == "ast.Exp.Lambda")
 
+    -- Check for duplicate parameter names.
+    --
+    -- Add each parameter name to the `names` set. If the current name already
+    -- exists in the set, report an error.
```

Comment:
We don't need to add a comment here because PR #222 got rid of this check for duplicate parameter names. I'm worried that git will gladly merge this comment into the master branch even though the block of code that it is commenting won't exist anymore.